customer-commons/src/main/java/no/unit/nva/customer/model/CustomerDto.java

```diff
@@ -478,6 +471,17 @@ public DoiAgentDto addId(URI doiAgentId) {
             return this;
         }
 
+        /**
+            Takes a customer identifier and build a doiAgent Id.
+         */
+        public DoiAgentDto addIdFromUuid(UUID identifier) {
```

Comment:
createIdFromUuid?

Reply:
addIdentifierAsId ?

the Id is a URI, this function convert an indentifier (UUID) into an Id